lib/correios/cep/address_finder.rb

```diff
@@ -19,6 +22,21 @@ def web_service
       def parser
         @parser ||= Correios::CEP::Parser.new
       end
+
+      private
+
+      def format_response(response)
```

Comment:
This method do not really format the response, after its execution response remains the same. A better name is desirable.